### DISCUSSION TITLE
Overpass in nominatim?

### DISCUSSION THREAD

**Discussion by marlin-mixon** [mediagis/nominatim-docker#567] [Q&A] [answered]
During the build, I noticed use of Overpass.  Is overpass exposed on the Nominatim server?  I tried the following:
curl -G "http://localhost:8080/api/interpreter" --data-urlencode "data=[out:json];node(50.745,7.17,50.75,7.18);out body;"
But I got a 404 error.
Thanks!  Marlin

**Comment by mtmail** [comment, marked as answer]
It’s not part of the build.

**Reply by marlin-mixon** [reply]
Oh, my mistake, I think I saw osm2pgsql and assumed that was an overpass reference.  Thanks!
